tests/js/server/replication/sync/replication-sync.js

```diff
@@ -2321,9 +2321,11 @@ function ReplicationNonIncrementalKeyConflict () {
 // / @brief executes the test suite
 // //////////////////////////////////////////////////////////////////////////////
 
-jsunity.run(ReplicationSuite);
-jsunity.run(ReplicationOtherDBSuite);
-jsunity.run(ReplicationIncrementalKeyConflict);
-jsunity.run(ReplicationNonIncrementalKeyConflict);
+if (internal.debugCanUseFailAt()) {
+  jsunity.run(ReplicationSuite);
+  jsunity.run(ReplicationOtherDBSuite);
+  jsunity.run(ReplicationIncrementalKeyConflict);
+  jsunity.run(ReplicationNonIncrementalKeyConflict);
+}
```

Comment:
```suggestion
```

I think excluding only the first two suites is enough.
Reading the file they seem to attempt to check themselves if debugFailAt is actually activated. Which for some reason does not work in all cases.